heroku/heroku-vscode
# Resource Explorer: Display Dynos

This PR adds the ability to view dynos associated with an app and the ability to stop or restart them from quick actions button.

## To test:
1. Pull down this branch, install deps run the extension
2. Open a heroku app project from the vscode instance running this extension.
3. Make sure you have a heroku remote defined `git remotes -v | grep heroku`
4. Login if you haven't already. 

You should see something similar to this screenshot:
<img width="419" alt="image" src="https://github.com/user-attachments/assets/9c7f2429-99f1-4efb-aaec-8ff8e2d6ea68">
